dojo/tools/dockle/parser.py

```diff
@@ -0,0 +1,57 @@
+import json
+import hashlib
+from dojo.models import Finding
+
+
+class DockleParser(object):
+    """
+    A class that can be used to parse the Dockle JSON report files
+    """
+
+    # table to match Dockle severity to DefectDojo severity
+    SEVERITY = {
+        "INFO": "Low",
+        "WARN": "Medium",
+        "FATAL": "High",
+    }
+
+    def get_scan_types(self):
+        return ["Dockle Scan"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return "Dockle Scan"
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Import JSON output for Dockle scan report."
+
+    def get_findings(self, filename, test):
+        data = json.load(filename)
+        dupes = {}
+        for item in data['details']:
+            code = item['code']
+            dockle_severity = item['level']
+            title = item['title']
+            if dockle_severity == "IGNORE":
+                continue
+            severity = self.SEVERITY[dockle_severity]
```

Comment:
this could lead to "key value" errors.
```suggestion
            if dockle_severity in self.SEVERITY:
                severity = self.SEVERITY[dockle_severity]
            else:
                severity = "Medium"
```